pkg/cmd/component/cmd/ui/ui.go

```diff
@@ -403,3 +403,81 @@ func AddEnvironmentName() string {
 	handleError(err)
 	return strings.TrimSpace(environmentName)
 }
+func ApplicationOutputPath() string {
+	var path string
+	prompt := &survey.Input{
+		Message: "Provide a path to application folder",
```

Comment:
Provide a path to the application folder

Reply:
Okay I will make the changes